frontend/src/components/eam-configurate-group/eam-configurate-group.tsx

```diff
@@ -85,6 +88,7 @@ const EAMConfigurateGroup: FC = () => {
         permissionsIds: selectedPermissions.selectedItems,
       };
       dispatch(EAMGroupConfigurateActions.update(newPayload));
+      dispatch(authActions.loadCurrentUser());
```

Comment:
I'd also add following code to the [update group action](https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/blob/dab2018f7466c254493ed49021eb0822b3c09dab/frontend/src/store/eam-group-configurate/actions.ts#L63)
```
const user = await authApi.getCurrentUserPermissions(); //TODO: add route on the backend

  const hasEamPermission = checkHasPermission(
    [Permission.MANAGE_EAM],
    auth,
  );

  if (hasEamPermission) {
    navigation.push(AppRoute.EAM);
    notification.success(
      NotificationTitle.SUCCESS,
      NotificationMessage.EAM_GROUP_UPDATE,
    );
  } else {
    navigation.push(AppRoute.ROOT);
  }